MapboxCoreNavigation/TunnelIntersectionManager.swift

```diff
@@ -108,22 +99,19 @@ open class TunnelIntersectionManager: NSObject {
         self.animatedLocationManager?.delegate = routeController
         self.animatedLocationManager?.routeProgress = routeProgress
         self.animatedLocationManager?.startUpdatingHeading()
-        
```

Comment:
@1ec5 we need to also add `self.animatedLocationManager?.startUpdatingLocation()` it got removed here.
https://github.com/mapbox/mapbox-navigation-ios/pull/1397/files#diff-9bf14ebab2a9283285d3a8e91bd09e86L54

Reply:
Done in d3df73b9a86ef6e9b19ea1af46299bf4b56ff495.